src/main/java/io/spring/releasenotes/properties/ApplicationProperties.java

```diff
@@ -53,6 +53,11 @@ public Github getGithub() {
 	 */
 	public static class Github {
 
+		/**
+		 * The base url for the github api. https://api.github.com is the default when not specified.
```

Comment:
Configuration properties metadata do not start with "The", "A", etc. Besides there is no need to document the default as the very purpose of this field-level javadoc is to complete some metadata that is extracted from the code (that includes the default value on the field).

Reply:
I removed the documentation of the default value.